src/applications/vaos/actions/newAppointment.js

```diff
@@ -300,7 +304,7 @@ export function openSelectAppointmentPage(page, uiSchema, schema) {
         if (dateObj.isAfter(now)) {
           acc.push({
             date: dateObj.format('YYYY-MM-DD'),
-            datetime: dateObj.format(),
+            datetime: dateObj.format('YYYY-MM-DD[T]HH:mm:ss'),
```

Comment:
@narin Without a format string, this was adding the local timezone offset at the end. Since all of these times are actually local to the facility, that offset is incorrect. This was causing problems later on when I tried to parse the date to submit.

Reply:
Good catch